addon/templates/components/frost-radio-button.hbs

```diff
@@ -1,10 +1,13 @@
 <label>
-  {{frost-radio-button-input
-    disabled=disabled
-    groupId=groupId
-    groupValue=groupValue
-    value=value
-  }}
+  <input
+    checked={{checked}}
+    class="frost-radio-button-input"
+    data-test={{hook (concat hook '-input')}}
```

Comment:
I reworked the hook so that if it is passed into the parent `radio-group` there is a concatenation with radio-group's hook and value to create the uniqueness that we want.